test/head_xpcshell_glue.js

```diff
@@ -1,173 +0,0 @@
-// This file needs to contain glue to rephrase the Mocha testsuite framework in
```

Comment:
While I do support adding the ability to run tests with Node.js via mocha, *removing* existing test suites is not a change I can support. The goal of this code is to run in multiple environments; the main reason I never tried to hard to get it working on Node.js is because Node was very bad about es6 support back when I started, and I haven't had the time to go back and fix issues (and also, it still doesn't support array destructuring or default parameters, which are just too damn useful. At least now I can go back and add classes everywhere!).